modules/light-clients/xx-mock/types/client_state.go

```diff
@@ -40,21 +42,22 @@ func (cs ClientState) GetLatestHeight() exported.Height {
 // GetTimestampAtHeight returns the timestamp in nanoseconds of the consensus state at the given height.
 func (cs ClientState) GetTimestampAtHeight(
 	ctx sdk.Context,
-	clientStore sdk.KVStore,
+	clientStore storetypes.KVStore,
 	cdc codec.BinaryCodec,
 	height exported.Height,
 ) (uint64, error) {
 	// get consensus state at height from clientStore to check for expiry
 	consState, found := getConsensusState(clientStore, cdc, height)
 	if !found {
-		return 0, sdkerrors.Wrapf(clienttypes.ErrConsensusStateNotFound, "height (%s)", height)
+		//return 0, sdkerrors.Wrapf(clienttypes.ErrConsensusStateNotFound, "height (%s)", height)
```

Comment:
Is there reason why this comment is left?

Reply:
https://github.com/dongrie/ibc-mock-client/commit/523df2202b5f4cc18db1d4048f4c5bfba4888817
Removed comment